IBERTpy/python/eyescan_plot.py

```diff
@@ -0,0 +1,268 @@
+# -*- coding: utf-8 -*-
```

Comment:
Maybe add the URL where you got this from (and make sure that you are respecting the license …)

Reply:
I added the URL to the original repository.